### ISSUE TITLE
Url from article does not link to current github project

### ISSUE THREAD

**Issue by darkBoft** [dotnet/AspNetCore.Docs#24256]
Hi,

in this Paragraph

```
For an overview of how to publish an ASP.NET Core web app, see Host and deploy ASP.NET Core. The MSBuild tasks and targets necessary to publish an ASP.NET Core web app are open-source in the dotnet/websdk repository.
```
The linked URL to "dotnet/websdk" repository does not link to the latest github repository.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 684d1f55-3d8b-9ba6-ee65-e8f84c2b4244
* Version Independent ID: 2db330b4-e830-715f-0a15-e406f85e1d3b
* Content: [Visual Studio publish profiles (.pubxml) for ASP.NET Core app deployment](https://docs.microsoft.com/en-us/aspnet/core/host-and-deploy/visual-studio-publish-profiles?view=aspnetcore-6.0)
* Content Source: [aspnetcore/host-and-deploy/visual-studio-publish-profiles.md](https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/host-and-deploy/visual-studio-publish-profiles.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-hostdeploy**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

**Comment by serpent5** [comment]
Hi @darkBoft. Thanks for reporting this.

It looks like the new URL should be https://github.com/dotnet/sdk/tree/main/src/WebSdk. Would you like to submit a PR to update it yourself?

**Comment by darkBoft** [comment]
Hi @serpent5 , let me contribute myself <3 thanks :)

**Comment by serpent5** [comment]
Thanks for choosing to contribute. Let me know here if you get stuck.

Once you submit your PR, re-open this issue. Be sure to include `Fixes #24256.` in the PR description, which will automatically close this issue once the PR is merged.
